charts/s1-agent/templates/helper/deployment.yaml

```diff
@@ -93,6 +93,8 @@ spec:
 {{- end }}
           - name: AGENT_CONTAINER_NAME
             value: "{{ include "agent.container_name" . }}"
+          - name: S1_INVENTORY_ONLY
```

Comment:
I think we should add it to the helper.config instead (_helpers.tpl)

{{- $_ := set $helperConfig "S1_INVENTORY_ONLY" (printf "%t" .Values.configuration.inventory_only) -}}

Reply:
Ok I'll do it after you merge your PR

Reply:
done